### PR TITLE
[Merged by Bors] - chore(Filter/Tendsto): split from `Basic`

### PR DESCRIPTION
**Pull request by urkud**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Order/Filter/Tendsto.lean [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+import Mathlib.Order.Filter.Basic
+
+/-!
+# Convergence in terms of filters
+
+The general notion of limit of a map with respect to filters on the source and target types
+is `Filter.Tendsto`. It is defined in terms of the order and the push-forward operation.
+The predicate "happening eventually" is `Filter.Eventually`, and "happening often" is
+`Filter.Frequently`, whose definitions are immediate after `Filter` is defined (but they come
+rather late in this file in order to immediately relate them to the lattice structure).
```
Comment (Ruben-VandeVelde) [inline Mathlib/Order/Filter/Tendsto.lean:15]: I think this part needs an update

File: Mathlib/Tactic/Peel.lean [hunk 2]
```diff
@@ -3,8 +3,8 @@ Copyright (c) 2023 Jireh Loreaux. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Jireh Loreaux
 -/
-import Mathlib.Order.Filter.Basic
 import Mathlib.Tactic.Basic
+import Mathlib.Order.Filter.Basic
```
Comment (Ruben-VandeVelde) [inline Mathlib/Tactic/Peel.lean:7]:
```suggestion
import Mathlib.Order.Filter.Basic
import Mathlib.Tactic.Basic
```

**Review by Ruben-VandeVelde — APPROVED** [review]
This makes sense to me, but please check the documentation

File: Mathlib/Order/Filter/Tendsto.lean (hunk 1 above)
Comment (urkud) [reply Mathlib/Order/Filter/Tendsto.lean]: Moved back to `Basic`.

**Review by grunweg — COMMENTED** [review]
Makes sense, thanks!
maintainer merge

**Comment by mattrobball** [comment]
bors merge

**Pull request closed without merging**